resources/application-connector/charts/connector-service/files/certs-restore.sh

```diff
@@ -0,0 +1,23 @@
+set -e
+set -o pipefail
+
+restore() {
+  namespace="$1"
+  secretName="$2"
+  backupSecretName="$secretName-backup"
+
+  echo "Restoring $backupSecretName"
+  secret=$(kubectl -n "$namespace" get secret "$backupSecretName" --ignore-not-found)
```

Comment:
You can easily replace the code with just one line` delete secret $backupSecretName" --ignore-not-found`